frontend/src/features/properties/map/activity/detail/ActivityForm.tsx

```diff
@@ -49,6 +49,7 @@ export const ActivityForm: React.FunctionComponent<IActivityFormProps> = ({
     <Formik<Api_Activity>
       initialValues={activity ?? { description: '', activityDataJson: '' }}
       onSubmit={async (values, formikHelpers) => {
+        values.activityStatusTypeCode = { id: values.status };
```

Comment:
we have a helper to assist with this - **toTypeCode** or something like that

Reply:
that's helpful, done